README.md

```diff
@@ -1 +1,82 @@
-# hello_microverse
+📗Table of Contents
+
+📖 About the Project
+🛠 Built With
+Tech Stack
+Key Features
+💻 Getting Started
+Prerequisites
+Setup
+Usage
+👥 Authors
+🤝 Contributing
+🙏 Acknowledgements
+📝 License
+
+📖 [Hello Microverse]
+
```

Comment:
- [ ] Please make sure that your Table of the content section of your readme file is linked to each section of your readme file. You can achieve that by following the [ README TEMPLATE](https://github.com/microverseinc/readme-template)
- [ ] I guess you copy the preview of the [README TEMPLATE](https://github.com/microverseinc/readme-template) instead of copying the raw view and customizing it to match your project. :smile: